src/app-layout/visual-refresh/app-bar.scss

```diff
@@ -89,13 +89,22 @@ section.appbar {
     }
 
     /*
-    If the content type supports a dark sticky header then 
-    minimal bottom padding should be on the breadcrumbs to
+    Table and cards content types implement a custom dark header. 
```

Comment:
The header is actually not dark for cards nor for table; this must have been true before Visual Refresh where this element was light gray, but now it is white like the rest of the content for both components.

Reply:
> ...since the header is actually not dark for cards nor for table...

The content type property is coupled with the full page variants of both components which have a persistent dark header. Unless I am missing something? If so could you elaborate...

Reply:
Here is the full page table demo: https://cloudscape.design/examples/react/table.html
And the full page cards demo: https://cloudscape.design/examples/react/cards.html

Reply:
Oh OK, it's about the full page versions, this is what I was missing.